frontend/locales/en-US.json

```diff
@@ -559,4 +563,4 @@
   "year": "Year",
   "years": "Years",
   "youMayAlsoLike": "You may also like"
-}
+}
```

Comment:
Your editor should've added a newline here, but seems it didn't. Please do it, since I'm not sure if it messes with Weblate